src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## **Unreleased**
 * Loosen Newtonsoft.JSON minimum version requirement from v13.0.1 to v9.0.1 [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/9.0.1). [#157](https://github.com/microsoft/jschema/pull/157)
+* FEATURE: Add new option `--generate-json-integer-as = int | long | biginteger | auto` and default as `int`. [#158](https://github.com/microsoft/jschema/pull/158)
```

Comment:
>Add new option `--generate-json-integer-as = int | long | biginteger | auto` and default as `int`. [#158](https://github.com/microsoft/jschema/pull/158) [](http://example.com/codeflow?start=11&length=152)

Please use this:

Add new option for specifying .NET type to express Json integers: `--generate-json-integer-as = int | long | biginteger | auto` with a default of `int`. [#158](https://github.com/microsoft/jschema/pull/158)

Reply:
Nice command-line argument name, by the way, very descriptive.

Reply:
Updated.